reV/SAM/generation.py

```diff
@@ -537,22 +523,30 @@ def set_nsrdb(self, resource):
         self['solar_resource_data'] = resource
 
 
-class Pvwatts(Solar, ABC):
-    """Photovoltaic (PV) generation with pvwattsv5.
+class AbstractPv(Solar, ABC):
```

Comment:
why AbstactPv instead of AbstractPV, I know its not perfect. camel case but Pv  is weird...

Reply:
Pv is weird but almost all of these classes have abbreviations in their name anyway... I would rather be rigorous about camel case vs. choosing what abbreviations deserve to be camel case. 

On the same note, i really regret having reV and SAM "properly" capitalized everywhere. I think we should either adhere to python cap convention or not at all. but those changes are probably too big to be really worth it right now.

Reply:
Thats fair, but on that point can we change the following:
Pvwatts* -> PvWatts*
Pvsam1 -> PvSam1

Reply:
agreed!